src/main/java/org/datadog/jenkins/plugins/datadog/model/DatadogLinkAction.java

```diff
@@ -0,0 +1,70 @@
+package org.datadog.jenkins.plugins.datadog.model;
+
+import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+import hudson.model.Action;
+import org.datadog.jenkins.plugins.datadog.util.conversion.DatadogConverter;
+import org.datadog.jenkins.plugins.datadog.util.conversion.VersionedConverter;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import static org.datadog.jenkins.plugins.datadog.util.conversion.VersionedConverter.ignoreOldData;
+
+public class DatadogLinkAction implements Action {
+
+    private final String url;
+
+    public DatadogLinkAction(BuildData buildData, String siteName) {
+        String query = String.format("ci_level:pipeline @ci.pipeline.name:\"%s\" @ci.pipeline.number:%s", buildData.getJobName(), buildData.getBuildNumber(""));
+        String urlEncodedQuery = URLEncoder.encode(query, StandardCharsets.UTF_8);
+        this.url = String.format("https://app.%s/ci/pipeline-executions?query=%s", siteName, urlEncodedQuery);
+    }
+
+    private DatadogLinkAction(String url) {
+        this.url = url;
+    }
+
+    @Override
+    public String getIconFileName() {
+        return "/plugin/datadog/icons/dd_icon_rgb.svg";
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "View in Datadog";
+    }
```

Comment:
Where this display is going to appear? Is it in the config section or in the pipeline detail?
I could not identify where it's used.

Reply:
It is in the pipeline detail
![image](https://github.com/user-attachments/assets/89c86369-fee6-405c-95cd-8001a8efe391)

Reply:
@sebastiaanspeck I have added an environment property (`DATADOG_JENKINS_PLUGIN_SHOW_DATADOG_LINKS`) and a UI control to the plugin config to disable this: https://github.com/jenkinsci/datadog-plugin/pull/495